style/checkStyle.xml

```diff
@@ -216,6 +216,12 @@
       <message key="name.invalidPattern"
                value="Member name ''{0}'' must match pattern ''{1}''."/>
     </module>
+    <module name="SuppressionXpathSingleFilter">
+      <property name="id" value="MemberName">
+      <property name="query" value="//ErrorElement[@sourceName='MemberName'][contains(@message, 'ConfigurationContextUtil.java')
+                                    or contains(@message, 'JsonUtils.java')
+                                    or contains(@message, 'RandomStringUtils.java')]"/>
+    </module>
```

Comment:
Exclusions like these are not very friendly to subsequent development and expansion. This will result in the need to configure the xml file whenever we need to add a new util class. I would suggest that for utility classes that do not apply to the `@UtilityClass` annotation, the annotation can be left out.